language/move-prover/bytecode/src/global_invariant_instrumentation_v2.rs

```diff
@@ -18,7 +18,7 @@ use crate::{
 use move_model::{
     ast::{ConditionKind, GlobalInvariant},
     exp_generator::ExpGenerator,
-    model::{FunctionEnv, GlobalEnv, GlobalId, QualifiedId, StructId},
+    model::{FunctionEnv, GlobalEnv, GlobalId, QualifiedInstId, StructId},
```

Comment:
@DavidLDill where we used QualifiedId<StructId> we are now using QualifiedInstId<StructId> to represent memory.

Reply:
Thanks.  That should be easy to deal with.